cmd/podman/tag.go

```diff
@@ -12,7 +12,7 @@ var (
 
 	tagDescription = "Adds one or more additional names to locally-stored image."
 	_tagCommand    = &cobra.Command{
-		Use:   "tag [flags] IMAGE TAG [TAG...]",
+		Use:   "tag [flags] IMAGE TARGET_NAME [TARGET_NAME...]",
```

Comment:
same comment here as in the other "tag" PR, `[flags]` here but no further word about them elsewhere?